src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -109,11 +109,6 @@ elseif(CLR_CMAKE_TARGET_LINUX)
     # 
     # # On OSX and *BSD, we use the libunwind that's part of the OS
```

Comment:
A nit - can you please delete the now empty if / endif and move the comment above to the location below where you have placed the libunwind finding?